<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Systemus::DataInterface</class>
 <widget class="QWidget" name="Systemus::DataInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QToolButton" name="nextPageButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>238</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>238</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="pageInput">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="specialValueText">
      <string>N / A</string>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="previousPageButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">&lt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="topBar">
     <item>
      <widget class="QLineEdit" name="searchInput">
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toggleFiltersButtons">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/filter_icon.png</normaloff>:/systemus/icons/filter_icon.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_refresh.png</normaloff>:/systemus/icons/action_refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_add.png</normaloff>:/systemus/icons/action_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_edit.png</normaloff>:/systemus/icons/action_edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="icon">
        <iconset resource="../resources/systemus_widgets_res.qrc">
         <normaloff>:/systemus/icons/action_delete.png</normaloff>:/systemus/icons/action_delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/systemus_widgets_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>previousPageButton</sender>
   <signal>clicked()</signal>
   <receiver>pageInput</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPageButton</sender>
   <signal>clicked()</signal>
   <receiver>pageInput</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
